common/src/app/common/data.cljc

```diff
@@ -477,6 +482,17 @@
   (->> (apply c/iteration args)
        (concat-all)))
 
+(defn insert-at-index
+  "Insert a list of elements at the given index of a previous list.
+  Replace all existing elems."
+  [elems index new-elems]
+  (let [[before after] (split-at index elems)
+        p? (set new-elems)]
```

Comment:
why `p?` ? It is a set, not a boolean

Reply:
this is the clojure idiom when we call a set as if it was a function, that returns true if the argument is inside the set